XRTK-Core/Packages/com.xrtk.core/Runtime/Utilities/BuildAndDeploy/UnityPlayerBuildTools.cs

```diff
@@ -185,6 +185,12 @@ public static async void StartCommandLineBuild()
             {
                 SyncSolution();
 
+                if (EditorUserBuildSettings.activeBuildTarget == BuildTarget.Android)
```

Comment:
Is this file really supposed to be in the runtime assembly?

Reply:
it's not. but we're refactoring the build tools after the 0.2 release.